server/src/main/java/org/elasticsearch/action/search/SearchPhaseExecutionException.java

```diff
@@ -161,4 +161,8 @@ public String toString() {
     public String getPhaseName() {
         return phaseName;
     }
+
+    public ShardSearchFailure[] getShardFailures() {
```

Comment:
I think this could be package scoped?

Reply:
It looks like there is already a method `shardFailures()` which I didn't see...so I will use the existing one and get rid of the one I added.